scene/gui/control.cpp

```diff
@@ -480,10 +480,10 @@ void Control::_notification(int p_notification) {
 			if (is_set_as_toplevel()) {
 				data.SI=get_viewport()->_gui_add_subwindow_control(this);
 
-				/*if (data.theme.is_null() && data.parent && data.parent->data.theme_owner) {
+				if (data.theme.is_null() && data.parent && data.parent->data.theme_owner) {
 					data.theme_owner=data.parent->data.theme_owner;
 					notification(NOTIFICATION_THEME_CHANGED);
-				}*/
+				}
```

Comment:
Thats what i did at first , but since it occasionally caused some glitches while testing which i was unable to resolve i reverted it to this one, which proven to works fine.

I disliked this bug very much and since no one tried to fix it for 2 months, it forced me to find it.
Still this is my first jump in, and i dont know godots internalls that good yet, so if you like, please feel free to try out if you can implement it properly, because it caused glitches when i tried.